src/wallet/wallet.cpp

```diff
@@ -6271,14 +6310,53 @@ KeyAddResult AddSpendingKeyToWallet::operator()(const libzcash::SaplingExtendedS
     }
 }
 
-std::optional<libzcash::UnifiedAddress> LookupUnifiedAddress::operator()(const libzcash::SaplingPaymentAddress& saplingAddr) const {
+// UFVKForReceiver :: (CWallet&, Receiver) -> std::optional<ZcashdUnifiedFullViewingKey>
```

Comment:
Why does this include CWallet& in the function argument?

Reply:
It's passed in the UFVKForReceiver constructor. So, partial application that returns the actual multiple-dispatch function that we're concerned with. I was merciful in that I didn't write it `CWallet& -> Receiver -> ...` :)